semantic-core/test/Spec.hs

```diff
@@ -95,12 +90,12 @@ parserSpecs = testGroup "Parsing: simple specs"
   , testCase "lambda with ASCII syntax" assert_ascii_lambda_parse
   , testCase "lambda with unicode syntax" assert_unicode_lambda_parse
   , testCase "quoted names" assert_quoted_name_parse
-  , testCase "let + dot precedence" assert_let_dot_precedence
-  , testCase "let in push" assert_let_in_push_precedence
   ]
 
 assert_roundtrips :: File (Term Core User) -> Assertion
-assert_roundtrips (File _ core) = parseEither Parse.core (showCore core) @?= Right (stripAnnotations core)
+assert_roundtrips (File _ core) = case parseEither Parse.core (showCore core) of
+  Right v -> v @?= stripAnnotations core
+  Left e -> assertFailure e
```

Comment:
This gives us better error messages, tho sadly I can’t seem to find a way to do the same in `tripping`.